src/components/ui/Breadcrumb/Breadcrumb.tsx

```diff
@@ -1,8 +1,19 @@
-import { Breadcrumb as UIBreadcrumb } from '@faststore/ui'
 import React, { memo } from 'react'
-import Icon from 'src/components/ui/Icon'
+import {
+  Breadcrumb as UIBreadcrumb,
+  Dropdown as UIDropdown,
+  DropdownButton as UIDropdownButton,
+  DropdownItem as UIDropdownItem,
+  DropdownMenu as UIDropdownMenu,
+} from '@faststore/ui'
 import Link from 'src/components/ui/Link'
 import type { BreadcrumbProps as UIBreadcrumbProps } from '@faststore/ui'
+import Icon from 'src/components/ui/Icon'
+import { navigate } from 'gatsby'
+
+import './breadcrumb.scss'
+
+const NOTEBOOK_WIDTH = 1280
```

Comment:
What do you think of reading this information at runtime from the CSS custom property `--fs-grid-breakpoint-notebook`? We did this at some point, but I think it got removed when fixing that hydration problem... Now I wonder if it's okay to use this approach again.

If that doesn't work, maybe we could move this to a more central place like `constants.ts` because future components may need it.

Reply:
I resolved this problem calling two breadcrumbs and controlling with "hidden-mobile" and "display-mobile", is bad because it is rendering two components, but was the only way which I found